cmd/sovereignnode/config/enableEpochs.toml

```diff
@@ -92,7 +92,7 @@
     ValidatorToDelegationEnableEpoch = 0
 
     # WaitingListFixEnableEpoch represents the epoch when the 6 epoch waiting list fix is enabled
-    WaitingListFixEnableEpoch = 1000000
+    WaitingListFixEnableEpoch = 0
```

Comment:
This flag I think should always be disabled or never used.

I think in the original `enableEpochs.toml` from `node` this flag should have been deleted.
Also, not sure about `CorrectLastUnjailedEnableEpoch` &  `BuiltInFunctionOnMetaEnableEpoch`. We should ask @sasurobert 's opinion as well

Reply:
- Deleted `WaitingListFixEnableEpoch` and `BuiltInFunctionOnMetaEnableEpoch`.
- I kept the `CorrectLastUnjailedEnableEpoch` because is a flag in enable epochs handler